app/design/frontend/Magento/blank/web/css/source/_navigation.less

```diff
@@ -135,8 +135,15 @@
         .switcher-dropdown {
             .lib-list-reset-styles();
             padding: @indent__s 0;
+            display: none;
```

Comment:
Are you sure that we should manually hide dropdown?

Reply:
Yes, we should for mobile because that will open if user will click in hi label.